### PR TITLE
publish 0.7.0

### PR DESCRIPTION
**Pull request by keertip**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-## unreleased
+## 0.7.0
+* [bug] show type information for Map with generics
```
Comment (sethladd) [inline CHANGELOG.md]: can you also mention that users can use the new crossdart option?

Comment (sethladd) [inline CHANGELOG.md]: https://github.com/dart-lang/dartdoc/pull/856

Comment (keertip) [inline CHANGELOG.md]: Yes, will do. Sorry missed that.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-## unreleased
+## 0.7.0
+* [feature] flag to add links to http://crossdart.info in the source code section
```
Comment (sethladd) [inline CHANGELOG.md]: Sorry to nit, but if you can, please add `--add-crossdart`

**Comment by sethladd** [comment]
lgtm with note

Thanks!

**Comment by keertip** [comment]
closes #935

**Pull request merged**